pages/pipelines/artifacts.md.erb

```diff
@@ -64,6 +64,20 @@ The `buildkite-agent artifact download` command only works within the context of
 
 If you want to download an artifact from outside a build use our [Artifact Download API](/docs/agent/v3/cli-artifact#downloading-artifacts-outside-a-running-build).
 
+## Encryption and security  
+
+Buildkite has zero access to your source code in the pipelines and only receives and stores the log output of the builds and build artifacts in encrypted form. 
+
+!!!
+
+Logs are AES-encrypted, and the build artifacts are encrypted in transit and at rest using AWS encryption (KMS or S3 SSE). As a result, the keys cannot be extracted by an insider, and the AWS solutions provide mitigations against zero-day attacks and other security issues. Beyond this, the control over security measures within your infrastructure is up to you.
+
+If you choose to [host your own build artifacts](/docs/agent/v3/cli-artifact#using-your-private-aws-s3-bucket), they never leave your infrastructure and end up in your private AWS bucket.
```

Comment:
Is it normal to consider your AWS as "your infrastructure"?

Reply:
Addressed this in the update.